docs/001_develop/03_client-capabilities/008_interaction/checkbox.mdx

```diff
@@ -0,0 +1,168 @@
+sidebar_label: 'Checkbox'
+id: client-interaction-checkbox
+keywords: [interaction, interactive, checkbox, box, boolean, form, click, change, checked, toggle, submit, data]
+tags:
+- interaction
+- interactive
+- checkbox
+- box
+- boolean
+- form
+- click
+- change
+- checked
+- toggle
+- submit
+- data
+sidebar_position: 8
+---
+
+import CheckboxDemo from '../../../../examples/ui/client-capabilities/interaction/checkbox.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Checkbox
+
+A form-connected implementation of a [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) input.
+
+## Example
+
+<CheckboxDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
+```
+Usage
+```typescript
+@customElement({
+    name: 'my-element',
+    templte: html`
+        <rapid-checkbox
+            ?disabled="${(x) => x.disabled}"
+            ?checked="${sync((x) => x.isSelected, 'boolean')}"
+            @change="${(x) => x.checkChanged()}"
+        >Checkbox
+        </rapid-checkbox>
+    `,
+})
+export class MyElement extends GenesisElement {
+    @observable disabled = false;
+    @observable isSelected = false;
+    checkChanged() {
+        console.log(this.isSelected);
+    }
+}
+```
+
+</TabItem>
+<TabItem value="react">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
+```
+Usage
+```typescript
+export function MyComponent() {
+    const [isSelected, setIsSelected] = useState(false);
+    // TODO: import this listener!
+    const webComponentRef = useCustomEventListener('change', (event) => {
+        setIsSelected(event.target.checked);
+        console.log({isSelected});
+    });
+
+    return (
+        <div>
+            <rapid-checkbox ref={webComponentRef}></rapid-checkbox>
+        </div>
+    );
+}
+```
+</TabItem>
+<TabItem value="angular">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
+```
+
+Model
+```typescript my.component.ts
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+@Component({
+  selector: 'my-root',
+  templateUrl: './my.component.html',
+  styleUrls: ['./my.component.css'],
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  imports: [FormsModule],
+})
+
+export class AppComponent {
+  isSelected: boolean = false;
+  onCheckboxChange(event: Event) {
+    const target = event.target as HTMLInputElement;
+    this.isSelected = target.checked;
+    console.log(this.isSelected);
+  }
+}
+```
+View
+```html
+<rapid-checkbox
+name="isSelected"
+[checked]="isSelected"
+(change)="onCheckboxChange($event)"
+></rapid-checkbox>
+```
+</TabItem>
+</Tabs>
+
+## DOM API
+
+Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
```

Comment:
Would it be helpful if we included type as a column here like angular docs seem to https://docs.angularjs.org/api/ng/input/input%5Bcheckbox%5D
we could have similar small types which stand out like they do too to add a bit more colour to all the pages

Reply:
Done 64416eb15a95506f0f9b523479e96bfbac0f6fb5